src/components/MainNavBar/MainNavBar.tsx

```diff
@@ -27,7 +27,9 @@ const MainNavBar = () => {
         .then((value) => {
           setIsLoggedInToUMLS(true);
         })
-        .catch((err) => {});
+        .catch((err) => {
+          handleToast("danger", "Login Failed. Please sign in again.", true);
```

Comment:
The message is correct as per the ticket. was just wondering if it makes sense to have a failure message more specific to UMLS.

Reply:
@adongare Thanks a lot Ashok! I was wondering about that too, but decided to go with requirement.